index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-     <meta charset="utf-8">
+   <meta name = "viewport" content = "width=device-width, initial-scale = 1;"/>
     <title>Test Suite Template</title>
```

Comment:
- [Optional] Please next time give a well-befitting title for the particular project, something like `Hello Microverse` would be okay for this project.